include/zeek-compat.h

```diff
@@ -19,7 +19,42 @@
 #endif
 #endif
 
+//// Collect all the Zeek includes here that we need anywhere in the plugin.
+
+#if ZEEK_DEBUG_BUILD
+#ifndef DEBUG
+#define DEBUG
```

Comment:
Let's remember here whether we forced `DEBUG` and only `undef` it below if we did, e.g.,

```suggestion
#define SPICY_PLUGIN_DEBUG_DEFINED
#define DEBUG
```